plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeSecurityConfig.java

```diff
@@ -37,7 +37,7 @@ public void testDefaults()
     public void testExplicitPropertyMappings()
     {
         Map<String, String> properties = ImmutableMap.<String, String>builder()
-                .put("delta.security", "read-only")
+                .put("delta.security", "read_only")
```

Comment:
I would prefer that `-` is supported too.

Reply:
It is actually supported, have a look in `DeltaLakeSecurityModule`.

```
install(conditionalModule(
     DeltaLakeSecurityConfig.class,
     // imitate Airlift's enum matching
     security -> name.equalsIgnoreCase(security.getSecuritySystem().replace("-", "_")), module));
}
```

I had a try with different Enums implementing a single Interface but Airlift tries to instantiate the interface in that case, so that was a no go.

Reply:
"read-only" should work; "read_only" doesn't have to, IMO

Reply:
The docs specify otherwise. See https://trino.io/docs/current/connector/delta-lake.html#authorization-checks

It's probably not going to be widely used as this just has been introduced anyway. I'm fine changing it, just let me know.